src/serac/physics/utilities/finite_element_state.cpp

```diff
@@ -61,6 +65,29 @@ FiniteElementState& FiniteElementState::operator=(const double value)
   return *this;
 }
 
+double avg(const FiniteElementState& state)
+{
+  double global_sum, local_sum   = state.trueVec().Sum();
```

Comment:
Style quibble: I don't like multiple declarations on a single line. However, it passes style check so I can't complain too much.

Reply:
I don't sure care so I'll go with what makes @jamiebramwell happy...